.gitignore

```diff
@@ -1 +1,4 @@
+/dist/
+/*.egg-info/
+/.coverage
```

Comment:
I'm assuming that the leading `/` is intentional to prevent the pattern from matching `foo/dist/`, `foo/foobar.egg-info`, or `foo/.coverage`. Otherwise, the leading slash can be dropped.

Reply:
It is intentional. Well spotted!